src/fixup.js

```diff
@@ -151,7 +151,6 @@
     try {
       var currentSpec = JSON.parse(request.responseText);
     } catch (err) {
-      console.error(err);
```

Comment:
Similarly here... we should never get to this code with anything but something in the 200 or 300 range (because of the if statement above), which makes the response "ok". The try/catch would only ever guard against an "ok" response that for whatever reason contained malformed data. 

Can you help me understand what happened?